scalameter-core/src/main/scala/org/scalameter/InvocationCount.scala

```diff
@@ -3,29 +3,24 @@ package org.scalameter
 import java.io.File
 import org.scalameter.Key._
 import org.scalameter.execution.invocation.InvocationCountMatcher
-import org.scalameter.execution.invocation.instrumentation.MethodInvocationCounter
-import org.scalameter.execution.invocation.instrumentation.MethodSignature
-import org.scalameter.execution.invocation.instrumentation.Instrumentation
-import org.scalameter.utils.ClassPath
-import scala.collection.Seq
-import scala.collection.mutable
+import org.scalameter.execution.invocation.instrumentation.{Instrumentation, MethodInvocationCounter, MethodSignature}
+import scala.collection.{Seq, mutable}
 
 
 /** Mixin for all [[org.scalameter.Measurer]] implementations that perform any kind of
  *  method invocation counting.
  */
-trait InvocationCount extends Measurer {
+trait InvocationCount extends Measurer[Double] {
```

Comment:
Yes, the plan is to make it in the next PR, although I suppose that it would be handy also if we have the reporter that collects such map result in html table? (I didn't start working on it yet)